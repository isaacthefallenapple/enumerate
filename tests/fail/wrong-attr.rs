use enumerate::Enumerate;

fn main() {
    #[derive(Enumerate)]
    enum Enum {
        A,
        B,
        #[enumerate(wrong attr)]
        C,
    }
}
