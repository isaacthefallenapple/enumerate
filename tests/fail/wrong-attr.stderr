error: Unexpected attribute: `wrong attr`
 --> $DIR/wrong-attr.rs:8:21
  |
8 |         #[enumerate(wrong attr)]
  |                     ^^^^^^^^^^
